docs/user-guide/global-configurations/user-access.md

```diff
@@ -36,23 +76,29 @@ When you click on Add User, you will see 5 options to set permission for users w
 
 In the `Email address` box, you have to provide the mail ID of the user to whom you want to give access to your applications.
 
+**`IMP`** Please note that Email address should be same as that in the `email` field in the JWT token returned by OIDC provider.
```

Comment:
user might have no clue about this. how user will verify if he is adding correct email ?

Reply:
I can't think of any way to tell user, except pointing them to documentation of dex - https://dexidp.io/docs/custom-scopes-claims-clients/. In case of github, it didnt return email Id but username if i remember correctly.